prov/efa/src/rdm/efa_rdm_pke_cmd.c

```diff
@@ -713,7 +713,13 @@ void efa_rdm_pke_handle_rx_error(struct efa_rdm_pke *pkt_entry, int prov_errno)
 			"Packet receive error from non TX/RX packet.  Our address: %s\n",
 			ep_addr_str);
 
-		efa_base_ep_write_eq_error(&ep->base_ep, err, prov_errno, true);
+		if (efa_rdm_pkt_type_of(pkt_entry) == EFA_RDM_WRITE_RTA_PKT ||
```

Comment:
Any rx error are unexpected, and I doubt whether you can really derive that pkt type here since the rx is failed.

Reply:
We need the RX error to be handled for 1 sided emulated atomics.